api/src/main/java/com/cloud/storage/StorageService.java

```diff
@@ -104,4 +104,6 @@
 
     ImageStore updateImageStoreStatus(Long id, Boolean readonly);
 
+    void syncHardwareCapability(long poolId);
```

Comment:
use generic method here, instead hard coded to specific capability

Reply:
Done